chart/values.yaml

```diff
@@ -24,11 +24,11 @@ distributor:
       enabled: true                          # Enable connection via Nats queue group to support exactly-once message processing
 
 remoteControlPlane:
-  enabled: false                             # Enables remote execution plane mode
+  enabled: true                              # Enables remote execution plane mode
```

Comment:
I think we should remove the `enabled` flag. It's always enabled, and we require connection to the Rest API + token